<launch>
    <arg name="plot_gui" default='true' doc="Launch rqt_plot to plot error"/>

    <rosparam command="load" file="$(find tsim)/config/trect.yaml" />

    <node name="turtle_rect" pkg="tsim" type="turtle_rect" output="screen" />
    <node name="turtlesim_node" pkg="turtlesim" type="turtlesim_node" />
    <group if="$(eval arg('plot_gui') == true)">
        <node name="error_plot" pkg="rqt_plot" type="rqt_plot" args="pose_error/x_error pose_error/y_error pose_error/theta_error" />
    </group>

</launch>